@here/harp-mapview/test/stubProjection.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+    GeoCoordinates,
+    mercatorProjection,
+    Projection,
+    ProjectionType
+} from "@here/harp-geoutils";
+import * as THREE from "three";
+
+/**
+ * Creates a stub of the Mercator projection.
+ * @param sandbox Sinon sandbox to keep track of created stubs.
+ * @returns MercatorProjection stub.
+ */
+export function stubMercatorProjection(sandbox: sinon.SinonSandbox): Projection {
```

Comment:
why do you stub the projection? Why not use mercator, or even just the identity projection?

Reply:
good point, the identity projection is enough for what I need.